tests_integration/nativeapp/test_bundle.py

```diff
@@ -39,14 +53,14 @@ def template_setup(runner, project_directory, request):
             assert Path(deploy_root, "setup_script.sql").is_file()
             assert Path(deploy_root, "README.md").is_file()
 
-            yield project_root, runner, test_project
+            yield project_root, execute_bundle_command, test_project
```

Comment:
i'm not sure I follow this, you're invoking the command above, but also passing it to the test so it can be invoked later?

Reply:
ah yeah, i see that it's meant to run twice. this is fine then